CMakeLists.txt

```diff
@@ -399,6 +399,8 @@ endif()
 #       - RIPEMD
 #       - BLAKE2
 
+set(WOLFSSL_AESCFB_HELP_STRING "Enable wolfSSL AES-CFB support (default: disabled)")
```

Comment:
Can you remove AES-CFB from the TODO above?

Reply:
Done